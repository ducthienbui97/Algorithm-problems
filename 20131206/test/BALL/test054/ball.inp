7 477710 724831 452213 556547
795908 869009 292096 919076 79783 915246 930063 987654 807843 452168 766095 512242 879741 880083 900746 145333 50528 55107 334217 978624 933212 947833 796357 476278 876653 493720 345429 41535 269282 529291 266645 101242 36013 727746 497479 546048 541807 90474 722673 639632 162327 258094 15967 737667 896718 85392 442558 418071 318171 
899991 201701 725159 612428 264099 408414 586930 444316 152137 808035 504604 892290 970284 941402 542820 132219 667597 789582 591456 519691 619632 465224 993132 949488 729701 348653 81209 445871 66007 948006 235124 126076 499439 219527 692467 616473 574996 227039 171575 918530 640183 531870 557069 334685 259282 267268 579319 319923 373088 
314513 376284 529819 717268 18999 693387 466702 272236 491336 704757 993201 846359 270729 927848 316313 572490 262863 534887 136380 986175 115018 709813 103426 786493 997476 489141 256530 363461 831963 965109 746723 214118 546699 37487 332537 646098 47590 855179 241546 794247 936216 95888 361116 105847 745874 533656 339610 908590 338170 
842725 232661 575390 875422 517218 714562 4827 822902 724667 74995 201622 517784 388541 632792 996635 366028 191335 800293 283743 882939 376292 588229 652103 907042 269656 390775 718891 217785 180683 92568 681329 982008 931123 724331 386962 706576 725727 850331 552341 894615 762023 452959 835069 593656 86836 620451 725347 599993 290786 431012 461961 852974 583321 135365 263645 947672 852080 973076 631608 716057 360965 348137 516643 437738 43778 897738 571632 249756 154070 760933 497719 304942 978411 386928 128197 674870 230309 454561 829335 667363 978788 232328 130379 246774 888631 212654 652191 592756 633283 58840 793509 633557 360075 816508 859504 847968 966091 619560 783150 584497 156255 911392 411358 579510 267020 727243 96522 139917 549325 876380 132300 533777 911329 28677 461335 113371 161303 546749 202880 32040 832811 601709 910938 220496 574083 4295 246775 976309 354455 776739 994790 977905 880294 2874 576730 32588 780193 784100 596997 765656 722937 737247 737395 972954 910798 752702 673112 391543 466548 915019 608081 150944 867118 513683 515416 574 896412 691844 836758 359926 509549 242248 441647 504742 336075 829932 204643 814979 356876 103746 985480 672321 222308 35486 706643 950580 613834 375801 898915 522287 593330 333193 729298 536374 370714 54199 53821 608567 841918 499819 306402 790922 738696 265473 341106 360978 921726 240832 158092 583167 6923 93932 705926 287724 505036 428137 250404 75704 872263 21497 175996 744313 378018 567829 533724 580543 527404 762910 371201 428597 490461 591747 855895 512630 51861 555906 599967 705989 50291 263012 703097 872955 319528 102058 645213 85749 667116 752937 933867 543457 521478 703191 52165 152636 539920 41306 289419 39247 494863 833171 245650 433279 142415 748807 79382 991891 31874 343275 516884 340757 384745 883449 65776 322155 654052 636928 483884 418032 674204 691179 246522 372453 487967 938482 77675 17072 466564 524239 564917 40554 259638 274273 764153 672379 159508 734340 494540 856110 616097 201011 646012 968327 481944 821041 713314 500546 676809 155099 848187 487151 89071 991008 184169 325572 63717 650274 862886 276246 891006 809778 87378 156585 355593 103155 598688 475872 137467 23753 737149 797175 369209 122716 198000 103132 334212 786736 602816 597753 36528 384392 343089 218204 453740 426027 38020 192408 488364 125558 722775 144112 30292 117180 910667 