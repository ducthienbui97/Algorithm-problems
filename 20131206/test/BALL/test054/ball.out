11940966520327
